.github/workflows/run-e2e-tests.yml

```diff
@@ -0,0 +1,149 @@
+name: Run Terralab CLI e2e tests with BEE using GCP workspace
+on:
+  push:
+    branches:
+      - main
+      - TSPS-331_cli_e2e_test
+    paths-ignore:
+      - '*.md'
+  workflow_dispatch:
+    inputs:
+      custom-app-version:
+        description: 'custom image tag to specify which version of teaspoons to deploy, e.g. `0.0.81-66ceced`. defaults to dev template if not specified.'
+        required: false
+        type: string
+      e2e-test-version-ref:
+        description: 'Name of branch or commit hash in the dsp-reusable-workflows repo to use. defaults to main'
+        required: false
+        type: string
+      wdl-method-version:
```

Comment:
so i think this and `teaspoons-version` arent both necessary

down below where we cal the e2e workflow we have this line
```
wdl-method-version: '${{ needs.init-github-context-and-params-gen.outputs.teaspoons-version }}'
```

Reply:
ah thanks, good call

Reply:
I'll keep `wdl-method-version`, since the teaspoons version deployed in the BEE is determined by the `custom-app-version` input